src/api/app/controllers/message_controller.rb

```diff
@@ -17,12 +17,12 @@ def check_project_and_package
   end
 
   def list
-    if @package
-      @messages = @package.messages
+    @messages = if @package
```

Comment:
strange indentation, wouldn't it be better to have `if`, `elseif` and `else` aligned?